### ISSUE TITLE
Search field scrolls off-screen

### ISSUE THREAD

**Issue by PieterL75** [netbox-community/netbox#15816]
### Deployment Type

NetBox Cloud

### NetBox Version

v4.0-beta1

### Python Version

3.8

### Steps to Reproduce

Scroll down on a page that has many lines

### Expected Behavior

The searchbar remains accessible at all times

### Observed Behavior

The searchbar scrolls offscreen

**Comment by jeffgdotorg** [comment]
Hi Pieter, thanks for the report. I see some inconsistencies in your report that need correcting before we can attempt to reproduce this problem:
- 4.0 betas are not yet available on NetBox Cloud, so I think your deployment type is incorrect
- Python 3.8 is [not supported in NetBox 4.0](https://github.com/netbox-community/netbox/blob/feature/docs/installation/index.md#requirements), so if that's the version you're in fact using, we need to know whether you still see the problem with Python 3.10 or later.

**Comment by jeffgdotorg** [comment]
On second reading, this issue appears to describe a feature request rather than a bug. Sticky-scroll behavior for the search box was intentionally omitted from the initial 4.0 scope in the interest of velocity.

It's valuable to know that somebody is paying attention to that aspect of the UI. If you'll reopen this issue as a feature request, we'll get the work scheduled to bring that behavior back when capacity permits.

**Comment by jeremystretch** [comment]
> this is bug/oversight

It's not. It was a conscious decision not to use a fixed top menu.

> And I don't like that you all just close these reports, in stead of taking them serious and swap them to a 'feature request'.

Submitting a feature request requires different information not provided above, which is why it has a separate issue template. Bug reports are to be used only when addressing behavior that differs from what was intended by the developer, _not_ from what you would prefer.

> I try to help you, and point out something. Have it closed is really counter-helpfull and probably just stop me of submitting more things that are differente in v4 then in v3.

Please use the correct issue templates when doing so. Misusing the forms only adds to the ever-increasing burden of maintaining the project.

**Comment by PieterL75** [comment]
I don't see that this was mentioned in the UI overhaul..
https://github.com/netbox-community/netbox/issues/12128
so assuming it was an oversight or something that is not functioning as it was before 4.0

I'll submit FR for all other in the future (but please dont close if they are bugs :-)
